Adding GOMA Finance to TrustWallet

gomafi opened the pull request.

Cryptocool1 commented:
hello,
• What is the purpose of this token? How will it benefit Trust Wallet users?
 • How is the token being distributed? Are you conducting an airdrop?
 • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
 • Is the token listed on a centralized/decentralized exchange?
 • Is a Smart Contract audit completed? Where can we find this information?

gomafi commented:
Hi there CryptoA,

Thanks for responding, I will answer this immediately - I appreciate 
your consideration!

A few things i'd like to note, our focus is not a MEMECoin, it's a 
Hyper-deflationary Community token with a very strong focus on DeFi, 
Staking and NFTs.

We have two prominent partnerships right now:

1.DEFI: Torii.Finance, they will be hosting our first STAKING within a 
week.

2. NFTs: STEM, https://stem.xyz, we will be launching/airdropping NFTs 
to GOMA holders in August

There are a number of considerations:

WHAT IS THE BENEFIT TO TRUSTWALLET USERS:

1. It has a redistribution feature, users will receive more tokens a 
strading volume increases - 3% on every trade

2. It has a burn function, the value will increase as scarcity grows - 
3% on every trade

3. We have built an initial dashboard, version 0.7 of GOMA.fi 
https://GOMA.fi - this is currently being updated SPECIFICALLY to 
include TrustWallet RIGHT NOW in v0.8 which should be live by tomorrow 
(you can take a look at our community, TrustWallet is the next thing to 
be integrated for our community since it currentlly uses MetaMask only 
in 0.7)

4. On this dashboard, users will also be able to STAKE GOMA for partner 
tokens (first one is "TORII"/Torii Finance - which launches in less than 
a week).

5. We are also launching GOMAx in July, GOMAx allows for uses who own 
GOMA (staked OR in their wallet - IE in trust) to claim and purchase 
GOMAx and receive redistribution of BNB into their TrustWallet AS WELL 
AS redistribution of GOMA additionally.

_We have designed the process so that users can hold the token in their 
wallet, and our system will assess the number of tokens they have and 
their % yield OR reflection will be based on their holdings of GOMA 
(which will be inside the TRUST WALLET). The ratio adjusts accordingly 
to holdings, so we maximise the use case for GOMA by insisting that they 
HOLD[in their wallet] and continue to trade into their wallet to 
increase the reflection yield._

HOW IS THE TOKEN DISTRIBUTED:

It is already distributed - 30% of the total token allocation is already 
in the hands of our community, they have purchased it and are busy 
trading it now 
https://bscscan.com/token/0xab14952d2902343fde7c65d7dc095e5c8be86920

DID YOU DO AN ICO / Presale:

We did a token presale on DXSale - 300BNB hardcap, and we completed it.

_Proof: https://dxsale.app/app/pages/defipresale?saleID=4897&chain=BSC_

IS THE TOKEN LIST ON EXCHANGES:

Yes, Currently Pancakeswap. We are in talks with Hotbit and coinsbit at 
the moment for launch on CEX.

Proofs:

TRADING: 
https://exchange.pancakeswap.finance/#/swap?outputCurrency=0xAb14952d2902343fde7c65D7dC095e5c8bE86920

INFO: 
https://pancakeswap.info/token/0xab14952d2902343fde7c65d7dc095e5c8be86920

THE AUDIT HAS BEEN SCHEDULED, HOWEVER WE ARE IN A QUEUE with 
https://CoinFabric.com

FURTHER INFO:

GOMA Fi: https://GOMA.fi v0.7 (0.8 will include TrustWallet by tomorrow)

CoinMarketCap: https://coinmarketcap.com/currencies/goma-finance/

CoinGecko: https://www.coingecko.com/en/coins/goma-finance

🔥MORE THAN 656 (65.6%) trillion $GOMA tokens have been burned and out of 
circulation since launch - thisi means that almost all of the tokens are 
owned by HOLDERS

- we have: 14,411 REAL holders - real holders matter, we do not work 
like a fluff coin.

Twitter: https://twitter.com/GomaShibaToken - 12,900 (cant see exact 
number) REAL followers.

Telegram ENG - https://t.me/GomaShiba - 30,387 REAL Members... 12,144 
online right now.

Telegram Chinese - https://t.me/GomaShibaChinese - 6,734 REAL members... 
2815 online right now.

On 2021-07-14 06:43, CryptoA wrote:

> hello,
> * What is the purpose of this token? How will it benefit Trust Wallet 
> users?
> * How is the token being distributed? Are you conducting an airdrop?
> * Did you sold the tokens at an Initial Coin Offering (ICO) or a token 
> presale?
> * Is the token listed on a centralized/decentralized exchange?
> * Is a Smart Contract audit completed? Where can we find this 
> information?
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].

-- 

🚀$GOMA🚀

GOMA is a Hyper-Deflationary DeFi token giving back to holders through 
passive growth & burning, staking, reflection & NFTs.

🦎 GOMA on CoinGecko 🦎

https://www.coingecko.com/en/coins/goma-financ

💎GOMA on CoinMarketCap💎

https://coinmarketcap.com/currencies/goma-finance/

COMMUNITIES:

English https://t.me/GomaShiba

Chinese http://t.me/GomaShibaChinese

Links:
------
[1] 
https://github.com/trustwallet/assets/pull/11755#issuecomment-879821666
[2] 
https://github.com/notifications/unsubscribe-auth/AUZOQD7MDX2IGFGTJYV4G4DTXVZ57ANCNFSM5ALGBP7Q

Cryptocool1 commented:
can you please add links to your PR?
example: https://developer.trustwallet.com/add_new_asset#info-json-contents

gomafi commented:
Please see the attacked info.json please let me know if this is correct:

added Twitter, telegram, github coingecko, cointelegraph thanks!

On 2021-07-14 09:38, CryptoA wrote:

> can you please add links to your PR?
> example: 
> https://developer.trustwallet.com/add_new_asset#info-json-contents
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].

-- 

🚀$GOMA🚀

GOMA is a Hyper-Deflationary DeFi token giving back to holders through 
passive growth & burning, staking, reflection & NFTs.

🦎 GOMA on CoinGecko 🦎

https://www.coingecko.com/en/coins/goma-financ

💎GOMA on CoinMarketCap💎

https://coinmarketcap.com/currencies/goma-finance/

COMMUNITIES:

English https://t.me/GomaShiba

Chinese http://t.me/GomaShibaChinese

Links:
------
[1] 
https://github.com/trustwallet/assets/pull/11755#issuecomment-879949797
[2] 
https://github.com/notifications/unsubscribe-auth/AUZOQDYQ5OPK6JVFP7BOV3DTXWOPFANCNFSM5ALGBP7Q
{
    "name": "GOMA Finance",
    "type": "BEP20",
    "symbol": "GOMA",
    "decimals": 9,
    "website": "https://GomaToken.com",
    "description": "GOMA is a Hyper-Deflationary Community DeFi token foused on staking, reflection and NFTs.",
    "explorer": "https://bscscan.com/token/0xAb14952d2902343fde7c65D7dC095e5c8bE86920",
    "status": "active",
    "id": "0xAb14952d2902343fde7c65D7dC095e5c8bE86920",
    "links": [
        {
            "name": "github",
            "url": "https://github.com/GOMA-Finance"
        },
        {
            "name": "twitter",
            "url": "https://twitter.com/GomaShibaToken"
        },
        {
            "name": "telegram",
            "url": "https://t.me/GomaShiba"
        },
        {
            "name": "coinmarketcap",
            "url": "https://coinmarketcap.com/currencies/goma-finance/"
        },
        {
            "name": "coingecko",
            "url": "https://www.coingecko.com/en/coins/goma-finance"
        }

	]
}

Cryptocool1 commented:
> Please see the attacked info.json please let me know if this is correct: added Twitter, telegram, github coingecko, cointelegraph thanks!
> […](#)

sorry you didn't add the links to your info.json

gomafi commented:
So sorry I dont know what happened - please see the attached

On 2021-07-14 11:05, CryptoA wrote:

>> Please see the attacked info.json please let me know if this is 
>> correct: added Twitter, telegram, github coingecko, cointelegraph 
>> thanks!
>> ...
> 
> sorry, you didn't add the links to your info.json
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].

-- 

🚀$GOMA🚀

GOMA is a Hyper-Deflationary DeFi token giving back to holders through 
passive growth & burning, staking, reflection & NFTs.

🦎 GOMA on CoinGecko 🦎

https://www.coingecko.com/en/coins/goma-financ

💎GOMA on CoinMarketCap💎

https://coinmarketcap.com/currencies/goma-finance/

COMMUNITIES:

English https://t.me/GomaShiba

Chinese http://t.me/GomaShibaChinese

Links:
------
[1] 
https://github.com/trustwallet/assets/pull/11755#issuecomment-880019655
[2] 
https://github.com/notifications/unsubscribe-auth/AUZOQD57T5BKPJ7CO3RBZSTTXWYVDANCNFSM5ALGBP7Q
{
    "name": "GOMA Finance",
    "type": "BEP20",
    "symbol": "GOMA",
    "decimals": 9,
    "website": "https://GomaToken.com",
    "description": "GOMA is a Hyper-Deflationary Community DeFi token foused on staking, reflection and NFTs.",
    "explorer": "https://bscscan.com/token/0xAb14952d2902343fde7c65D7dC095e5c8bE86920",
    "status": "active",
    "id": "0xAb14952d2902343fde7c65D7dC095e5c8bE86920",
    "links": [
        {
            "name": "github",
            "url": "https://github.com/GOMA-Finance"
        },
        {
            "name": "twitter",
            "url": "https://twitter.com/GomaShibaToken"
        },
        {
            "name": "telegram",
            "url": "https://t.me/GomaShiba"
        },
        {
            "name": "coinmarketcap",
            "url": "https://coinmarketcap.com/currencies/goma-finance/"
        },
        {
            "name": "coingecko",
            "url": "https://www.coingecko.com/en/coins/goma-finance"
        }

	]
}

gomafi commented:
Am I misunderstanding you? If so, please let me know!

We will add whatever we need to

On 2021-07-14 11:05, CryptoA wrote:

>> Please see the attacked info.json please let me know if this is 
>> correct: added Twitter, telegram, github coingecko, cointelegraph 
>> thanks!
>> ...
> 
> sorry, you didn't add the links to your info.json
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].

-- 

🚀$GOMA🚀

GOMA is a Hyper-Deflationary DeFi token giving back to holders through 
passive growth & burning, staking, reflection & NFTs.

🦎 GOMA on CoinGecko 🦎

https://www.coingecko.com/en/coins/goma-financ

💎GOMA on CoinMarketCap💎

https://coinmarketcap.com/currencies/goma-finance/

COMMUNITIES:

English https://t.me/GomaShiba

Chinese http://t.me/GomaShibaChinese

Links:
------
[1] 
https://github.com/trustwallet/assets/pull/11755#issuecomment-880019655
[2] 
https://github.com/notifications/unsubscribe-auth/AUZOQD57T5BKPJ7CO3RBZSTTXWYVDANCNFSM5ALGBP7Q

cryptomanz commented:
you need to add links to your info.json
example: https://developer.trustwallet.com/add_new_asset#info-json-contents

gomafi commented:
Im very sorry, I cant see it? Do we need to upload to the same 
directory? or through email?

I added a .json file into the email? Or must we push via github? Will we 
pay AGAIN?

On 2021-07-14 12:49, Nazar wrote:

> you need to add links to your info.json
> example: 
> https://developer.trustwallet.com/add_new_asset#info-json-contents
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].

-- 

🚀$GOMA🚀

GOMA is a Hyper-Deflationary DeFi token giving back to holders through 
passive growth & burning, staking, reflection & NFTs.

🦎 GOMA on CoinGecko 🦎

https://www.coingecko.com/en/coins/goma-financ

💎GOMA on CoinMarketCap💎

https://coinmarketcap.com/currencies/goma-finance/

COMMUNITIES:

English https://t.me/GomaShiba

Chinese http://t.me/GomaShibaChinese

Links:
------
[1] 
https://github.com/trustwallet/assets/pull/11755#issuecomment-880090337
[2] 
https://github.com/notifications/unsubscribe-auth/AUZOQD3KN2ZSUEYBDL43A43TXXE4NANCNFSM5ALGBP7Q

cryptomanz commented:
> Im very sorry, I cant see it? Do we need to upload to the same directory? or through email? I added a .json file into the email? Or must we push via github? Will we pay AGAIN?
> […](#)

just edit the file, adding your links

gomafi commented:
I did three times :(

https://www.dropbox.com/s/cyr1lydkrrrfmnf/info.json?dl=0 <--- info.json

Where do I put it.. i've attached it again please see - if it doesnt 
attached then its a spamblock

On 2021-07-14 13:02, Nazar wrote:

>> Im very sorry, I cant see it? Do we need to upload to the same 
>> directory? or through email? I added a .json file into the email? Or 
>> must we push via github? Will we pay AGAIN?
>> ...
> 
> just edit the file, adding your links
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].

-- 

🚀$GOMA🚀

GOMA is a Hyper-Deflationary DeFi token giving back to holders through 
passive growth & burning, staking, reflection & NFTs.

🦎 GOMA on CoinGecko 🦎

https://www.coingecko.com/en/coins/goma-financ

💎GOMA on CoinMarketCap💎

https://coinmarketcap.com/currencies/goma-finance/

COMMUNITIES:

English https://t.me/GomaShiba

Chinese http://t.me/GomaShibaChinese

Links:
------
[1] 
https://github.com/trustwallet/assets/pull/11755#issuecomment-880098838
[2] 
https://github.com/notifications/unsubscribe-auth/AUZOQDZNKWMJFTSE5RXE6CLTXXGL7ANCNFSM5ALGBP7Q
{
    "name": "GOMA Finance",
    "type": "BEP20",
    "symbol": "GOMA",
    "decimals": 9,
    "website": "https://GomaToken.com",
    "description": "GOMA is a Hyper-Deflationary Community DeFi token foused on staking, reflection and NFTs.",
    "explorer": "https://bscscan.com/token/0xAb14952d2902343fde7c65D7dC095e5c8bE86920",
    "status": "active",
    "id": "0xAb14952d2902343fde7c65D7dC095e5c8bE86920",
    "links": [
        {
            "name": "github",
            "url": "https://github.com/GOMA-Finance"
        },
        {
            "name": "twitter",
            "url": "https://twitter.com/GomaShibaToken"
        },
        {
            "name": "telegram",
            "url": "https://t.me/GomaShiba"
        },
        {
            "name": "coinmarketcap",
            "url": "https://coinmarketcap.com/currencies/goma-finance/"
        },
        {
            "name": "coingecko",
            "url": "https://www.coingecko.com/en/coins/goma-finance"
        }

	]
}

gomafi commented:
This is what we have in that file:

{
"name": "GOMA Finance",
"type": "BEP20",
"symbol": "GOMA",
"decimals": 9,
"website": "https://GomaToken.com",
"description": "GOMA is a Hyper-Deflationary Community DeFi token foused 
on staking, reflection and NFTs.",
"explorer": 
"https://bscscan.com/token/0xAb14952d2902343fde7c65D7dC095e5c8bE86920",
"status": "active",
"id": "0xAb14952d2902343fde7c65D7dC095e5c8bE86920",
"links": [
{
"name": "github",
"url": "https://github.com/GOMA-Finance"
},
{
"name": "twitter",
"url": "https://twitter.com/GomaShibaToken"
},
{
"name": "telegram",
"url": "https://t.me/GomaShiba"
},
{
"name": "coinmarketcap",
"url": "https://coinmarketcap.com/currencies/goma-finance/"
},
{
"name": "coingecko",
"url": "https://www.coingecko.com/en/coins/goma-finance"
}

]
}

On 2021-07-14 13:02, Nazar wrote:

>> Im very sorry, I cant see it? Do we need to upload to the same 
>> directory? or through email? I added a .json file into the email? Or 
>> must we push via github? Will we pay AGAIN?
>> ...
> 
> just edit the file, adding your links
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].

-- 

🚀$GOMA🚀

GOMA is a Hyper-Deflationary DeFi token giving back to holders through 
passive growth & burning, staking, reflection & NFTs.

🦎 GOMA on CoinGecko 🦎

https://www.coingecko.com/en/coins/goma-financ

💎GOMA on CoinMarketCap💎

https://coinmarketcap.com/currencies/goma-finance/

COMMUNITIES:

English https://t.me/GomaShiba

Chinese http://t.me/GomaShibaChinese

Links:
------
[1] 
https://github.com/trustwallet/assets/pull/11755#issuecomment-880098838
[2] 
https://github.com/notifications/unsubscribe-auth/AUZOQDZNKWMJFTSE5RXE6CLTXXGL7ANCNFSM5ALGBP7Q

gomafi commented:
I just updated the info.json

On 2021-07-14 13:02, Nazar wrote:

>> Im very sorry, I cant see it? Do we need to upload to the same 
>> directory? or through email? I added a .json file into the email? Or 
>> must we push via github? Will we pay AGAIN?
>> ...
> 
> just edit the file, adding your links
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].

-- 

🚀$GOMA🚀

GOMA is a Hyper-Deflationary DeFi token giving back to holders through 
passive growth & burning, staking, reflection & NFTs.

🦎 GOMA on CoinGecko 🦎

https://www.coingecko.com/en/coins/goma-financ

💎GOMA on CoinMarketCap💎

https://coinmarketcap.com/currencies/goma-finance/

COMMUNITIES:

English https://t.me/GomaShiba

Chinese http://t.me/GomaShibaChinese

Links:
------
[1] 
https://github.com/trustwallet/assets/pull/11755#issuecomment-880098838
[2] 
https://github.com/notifications/unsubscribe-auth/AUZOQDZNKWMJFTSE5RXE6CLTXXGL7ANCNFSM5ALGBP7Q
